pyproject.toml

```diff
@@ -44,10 +44,10 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.8"
 numpy = [
-    { version = ">=1.21.2,<1.23.3", python = "<=3.10" },
-    { version = ">=1.23.3", python = ">3.10" },
+    { version = ">=1.21.2,<=2.1.3", python = "<3.13" },
```

Comment:
I suggest, if we don't impose an upper-bound for `>=3.13`, that we don't impose it for `<3.13`.